airbyte-workers/src/main/java/io/airbyte/workers/temporal/TemporalClient.java

```diff
@@ -137,4 +312,40 @@ public TemporalClient(final WorkflowClient client, final Path workspaceRoot) {
     return new TemporalResponse<>(operationOutput, metadata);
   }
 
+  /**
+   * Check if a workflow is currently running. It is using the temporal pagination (see:
+   * https://temporalio.slack.com/archives/CTRCR8RBP/p1638926310308200)
+   */
+  public boolean isWorkflowRunning(final String workflowName) {
+    ByteString token;
+    ListOpenWorkflowExecutionsRequest openWorkflowExecutionsRequest =
+        ListOpenWorkflowExecutionsRequest.newBuilder()
+            .setNamespace(client.getOptions().getNamespace())
```

Comment:
this feels less than 150 word wrap..

Reply:
It is likely that I have done it manually. I often work with a split window and the 150 chars wrap is a little bit too much for me. Is it an issue to not maximize the line size?

Reply:
not exactly. it's my personal preference - I find it easier to read the code when it's on the same line since I don't have to jump up and down to follow all the signature, the constructor and the builder methods.

I'm also used to indents as an indicator for block code execution (probably from working with space based languages) so at first glance this step ladder reads as 3 scopes to me.